docs/contributing.md

```diff
@@ -291,6 +291,10 @@ We recommend at least these settings:
 }
 ```
 
+### Alternative development setup
+
+An experimental configuration is available for [`mise`](https://mise.jdx.dev/).
+
```

Comment:
nit: expand this section a bit for common tasks

Reply:
I voluntarily only mentioned it so people can know it exists, but without anything more, while it's experimental.
There are a few commands listed in `mise.toml`.
Does that work for you?

Reply:
> I voluntarily only mentioned it so people can know it exists, but without anything more, while it's experimental.

yes, I know that but it still should mention the common flows that are used in the contributing doc with mise since the people who want use it to contribute should follow the same-ish patterns as the rest.

Reply:
I've added a few commonly used commands